src/platform/common/process/environmentActivationService.node.ts

```diff
@@ -60,7 +60,8 @@ export const defaultShells = {
     [OSType.Windows]: { shell: 'cmd', shellType: TerminalShellType.commandPrompt },
     [OSType.OSX]: { shell: 'bash', shellType: TerminalShellType.bash },
     [OSType.Linux]: { shell: 'bash', shellType: TerminalShellType.bash },
-    [OSType.Unknown]: undefined
+    [OSType.Unknown]: undefined,
+    [OSType.Web]: undefined
```

Comment:
Hmm, I think we might want OSWindowsWeb, OSLinuxWeb, OSMacWeb.
As OSWeb seems too ambiguous.
Then again, I guess we could always add that later.

Reply:
Because saying an OS is web feel very wrong, if anything is the UI/deployment/runtime kind

Reply:
I think we should discuss this, the more i think of this, I feel it could end up being one of those things that will cause confusions.

Reply:
It doesn't for me. When you're in a web extension you can't tell what the OS is. The only thing you can tell is that you're running in a browser.

Reply:
I removed the OS.Web type but used process.platform to determine if running in the web or not still.
Added another function to determine if 'isWeb' instead.